plugins/git/git.plugin.zsh

```diff
@@ -197,6 +197,7 @@ alias ggpur='ggu'
 alias ggpull='git pull origin "$(git_current_branch)"'
 alias ggpush='git push origin "$(git_current_branch)"'
 
+alias ggr='git grep -n --color=always'
```

Comment:
To keep it in sync with #11259, `-n` could be replaced with `--line-number`. Documentation [`git grep --line-number`](https://git-scm.com/docs/git-grep#Documentation/git-grep.txt--n)

Reply:
Also, [the `--color` option](https://git-scm.com/docs/git-grep#Documentation/git-grep.txt---colorltwhengt) defaults to `always`, so we don't need to provide it.